feature-libs/my-account/organization/components/budget/budget.config.ts

```diff
@@ -57,33 +56,40 @@ export const budgetRoutingConfig: RoutingConfig = {
 export const budgetCmsConfig: CmsConfig = {
   cmsComponents: {
     ManageBudgetsListComponent: {
-      component: BudgetListComponent,
+      component: OrganizationListComponent,
+      providers: [
+        {
+          provide: OrganizationListService,
+          useExisting: BudgetListService,
+        },
+        {
+          provide: OrganizationItemService,
+          useExisting: BudgetItemService,
+        },
+      ],
       childRoutes: [
         {
           path: 'create',
-          component: BudgetCreateComponent,
-          canDeactivate: [SplitViewDeactivateGuard],
+          component: BudgetFormComponent,
         },
         {
           path: `:${ROUTE_PARAMS.budgetCode}`,
           component: BudgetDetailsComponent,
           canActivate: [ExistBudgetGuard],
-          canDeactivate: [SplitViewDeactivateGuard],
           children: [
+            {
+              path: `edit`,
+              component: BudgetFormComponent,
+              canActivate: [ActiveBudgetGuard],
+            },
             {
               path: 'cost-centers',
               component: BudgetCostCenterListComponent,
-              canDeactivate: [SplitViewDeactivateGuard],
             },
           ],
         },
-        {
-          path: `:${ROUTE_PARAMS.budgetCode}/edit`,
-          component: BudgetEditComponent,
-          canActivate: [ActiveBudgetGuard],
-        },
       ],
-      guards: [AuthGuard, AdminGuard],
+      guards: [AuthGuard],
```

Comment:
Why did you decide to remove AdminGuard?

Reply:
I've overlooked that I'm afraid when I merged, thx! I'll put it back.